salt/monitoring/prometheus/rules.yml

```
@@ -26,8 +26,18 @@ groups:
     annotations:
       summary: A cluster resource failed
 
-# sbd alerts
 
+# failcount exceed migration threshold (example on saphana specific resource)
+- name: resource-failcount-higher-threshold
+  rules:
+  - alert: resource-failcount-higher-threshold
+    expr: ha_cluster_pacemaker_fail_count{resource="rsc_SAPHana_PRD_HDB00"} > ha_cluster_pacemaker_migration_threshold{resource="rsc_SAPHana_PRD_HDB00"}
```

here is the correct query we should use:
```suggestion
    expr: count(ha_cluster_pacemaker_fail_count > ha_cluster_pacemaker_migration_threshold) > 0
```